packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -138,17 +138,7 @@ function getBabelEsmPlugin(config) {
 					: '[name].esm.js',
 				chunkFilename: '[name].chunk.[chunkhash:5].esm.js',
 				excludedPlugins: ['BabelEsmPlugin', 'InjectManifest'],
-				beforeStartExecution: (plugins, newConfig) => {
-					const babelPlugins = newConfig.plugins;
-					newConfig.plugins = babelPlugins.filter((plugin) => {
```

Comment:
this makes sure `'fast-async'` are not added, why remove this?

Reply:
I did a serach for `fast-async` in code and figured if I'm removing it as dependency any related code would be safe to remove also. Unles goal was to also prevent users from adding it via custom `babelrc`?